From e53af8b890bed07f9cba5eed1ed52bc99bb7c306 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Fri, 20 May 2022 15:29:32 +0000
Subject: [PATCH] Change the memtrack aidl hal service name

After implement the memtrack aidl hal, change the service name.

Tracked-On: OAM-98918
Signed-off-by: HeYue <yue.he@intel.com>
---
 vendor/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/file_contexts b/vendor/file_contexts
index 9ea31c926..def774fd6 100644
--- a/vendor/file_contexts
+++ b/vendor/file_contexts
@@ -87,7 +87,7 @@
 /(vendor|system/vendor)/bin/hw/android\.hardware\.media\.c2-default-service   u:object_r:mediacodec_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.tv\.mediaquality-service\.example  u:object_r:hal_mediaquality_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.memtrack@1\.0-service       u:object_r:hal_memtrack_default_exec:s0
-/(vendor|system/vendor)/bin/hw/android\.hardware\.memtrack-service.example    u:object_r:hal_memtrack_default_exec:s0
+/(vendor|system/vendor)/bin/hw/android\.hardware\.memtrack-service.celadon    u:object_r:hal_memtrack_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.motioncontext-service       u:object_r:hal_motioncontext_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.motioncontext-service\.default u:object_r:hal_motioncontext_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.nfc@1\.0-service            u:object_r:hal_nfc_default_exec:s0
-- 
2.34.1

